README.md

```diff
@@ -16,7 +16,7 @@ For further developer and user documentation please visit [the wiki](https://git
 ##### Minimum PHP Version
 * PHP >= 5.4
 
-##### Minimum ownCloud Version
+##### Minimum Nextcloud Version
 * >= 8.1
```

Comment:
I've listed both for now. I can imagine Nextcloud Notes turning into a separate codebase with the upcoming Nextcloud version 11.